tst/testrandom.g

```diff
@@ -49,8 +53,22 @@ randomTest := function(collection, method, checkin...)
     fi;
 end;;
 
+randomTest := function(collection, method, checkin...)
+    local checkmethod;
+    if Length(checkin) = 0 then
+        checkmethod := \in;
+    else
+        checkmethod := checkin[1];
+    fi;
+
+    randomTestInner(IsMersenneTwister, GlobalMersenneTwister, x -> method(x), method, collection, checkmethod);
+    randomTestInner(IsGAPRandomSource, GlobalRandomSource, x -> method(GlobalRandomSource, x), method, collection, checkmethod);
+end;
+
 # A special test for collections of size 1
-randomTestForSizeOneCollection := function(collection, method)
+# Here we can't check different seeds produce different answers
+# We do check that the random source is not used, for efficency.
```

Comment:
While that seems sensible, is this documented anywhere?

Reply:
No, it seemed to be true in practice, so I decided to test it. I'm not positive I'd want to enforce it, but if it was ever false I'd certainly want to investigate why, as I imagine we'd be doing pointless work.